pyscript.core/src/stdlib/pyscript/magic_js.py

```diff
@@ -21,7 +22,6 @@ def current_target():
 
 else:
     import _pyscript
-    from _pyscript import PyWorker
```

Comment:
these kind of changes make we wish to remove any linter from our source code ... this effectively side-effects on the module as `PyWorker` won't exist anymore so if it's pre-commit doing this, please remove this file from its greedy and breaking changes, thank you.

Reply:
yes, I completely agree with you.
A linter which breaks our code is not a linter we can trust, so I'm going to remove ruff entirely.